test/evergreen.yml

```diff
@@ -1159,10 +1221,9 @@ variables:
       CC_OPTIMIZE_LEVEL: -DCC_OPTIMIZE_LEVEL=-O0
       smp_command: -j $(sysctl -n hw.logicalcpu)
       cmake_generator: "Unix Makefiles"
```

Comment:
I guess this line answers my question regarding `Ninja`. Do we know why we are using `Unix Makefiles` here? If this is because it is not available on macOS, fair enough, my comments can be resolved.

Reply:
I believe this is because the machine doesn't have ninja installed. @lukech  can you help confirm this?

Reply:
I don't have a recollection of this, and can see the rhel80 and amazon2-arm64 builders also use make instead of ninja, which is unlikely about package missing  (though could be copy-paste).

Reply:
One possibility is to maintain the testing diversity to have both covered.

Reply:
For this ticket we should focus on cleaning up the existing variables rather than adding more tests. I am happy to create a ticket to investigate.